.devcontainer/devcontainer.json

```diff
@@ -18,13 +19,20 @@
         "vscode": {
             // Add the IDs of extensions you want installed when the container is created.
             "extensions": [
-                "ms-dotnettools.csharp",
+                "github.vscode-github-actions",
+                "GitHub.copilot",
+                "GitHub.vscode-github-actions",
+                "ms-dotnettools.vscode-dotnet-runtime",
+                "ms-dotnettools.csdevkit",
+                "ms-dotnetools.csharp",
                 "Ionide.Ionide-fsharp",
                 "tintoy.msbuild-project-tools"
             ]
         }
     },
-
-    "onCreateCommand": [ "dotnet", "build", "FSharp.Compiler.Service.sln"],
-    "waitFor": "onCreateCommand"
+    "remoteEnv": {
+        "DOTNET_MULTILEVEL_LOOKUP": "0",
```

Comment:
This env var isn't required - multilevel lookup was never used on Linux, and in 7.0 we [disabled it by default on Windows as well](https://learn.microsoft.com/en-us/dotnet/core/compatibility/deployment/7.0/multilevel-lookup)

Reply:
I think I copied it over from some example.